tests/modules/nf-core/hisat2/build_test/main.nf

```diff
@@ -7,11 +7,33 @@ include { HISAT2_BUILD              } from '../../../../../modules/nf-core/hisat
 
 workflow test_hisat2_build {
     fasta = [ [id:'genome'],
-              file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+              file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
             ]
     gtf   = [ [id:'genome'],
-              file(params.test_data['sarscov2']['genome']['genome_gtf'], checkIfExists: true)
+              file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)
             ]
     HISAT2_EXTRACTSPLICESITES ( gtf )
     HISAT2_BUILD ( fasta, gtf, HISAT2_EXTRACTSPLICESITES.out.txt )
 }
+
+workflow test_hisat2_build_fasta_only {
+    fasta = [ [id:'genome'],
+              file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
+            ]
+    gtf   = [ [id:'genome'],
+              file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)
+            ]
+    HISAT2_EXTRACTSPLICESITES ( gtf )
```

Comment:
Not needed to be executed here if not passed to HISAT2_BUILD in this test (and `gtf` could be removed I guess)